source/frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -214,6 +224,46 @@ export const PropertyFilter: React.FC<React.PropsWithChildren<IPropertyFilterPro
                   innerClassName="flex-nowrap"
                 ></CoordinateSearchForm>
               )}
+              {values.searchBy === 'surveyParcel' && (
+                <Row noGutters>
+                  <Col>
+                    <Select
+                      innerClassName="rounded-0"
+                      field="district"
+                      options={LandTitleDistricts.map(ltd => ({
+                        value: ltd,
+                        label: ltd,
+                      }))}
+                    />
+                  </Col>
+                  <Col>
+                    <NumberInput
+                      innerClassName="rounded-0"
+                      placeholder="Section"
+                      field="section"
+                      onFocus={e => e.target.select()}
+                      displayErrorTooltips
+                    ></NumberInput>
+                  </Col>
+                  <Col>
+                    <NumberInput
+                      innerClassName="rounded-0"
+                      placeholder="Township"
+                      field="township"
+                      onFocus={e => e.target.select()}
+                      displayErrorTooltips
+                    ></NumberInput>
+                  </Col>
+                  <Col>
+                    <NumberInput
+                      placeholder="Range"
+                      field="range"
+                      onFocus={e => e.target.select()}
+                      displayErrorTooltips
+                    ></NumberInput>
```

Comment:
Just a question of whether **all** these fields are NUMERIC? I recall range having values like "5E" (which the number input would not support...)
A quick look at LTSA Surveyed Parcels layers shows that Range can be "5 EAST"
![image](https://github.com/user-attachments/assets/fa0e6bcc-c9a5-4cfc-b005-4dee213062c4)